setup_directories.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
```

Comment:
does this need to be python too? I didn't intend to use python other than for things that really use python libs/ML/AI